README_WINDOWS.rst

```diff
@@ -6,18 +6,20 @@ Build Requirements
 
 - VC 14.0 Compiler
 - OpenSSL Library (https://slproweb.com/products/Win32OpenSSL.html)
-- Additional Python Packages:
-
-  - psutils
-
 - Patience and a lot of coffee / beer
 
 What's working
 --------------
 
-- Borg does not crash if called with ``borg``
-- ``borg init --encryption none ./demoRepo`` runs without an error/warning.
-   Note that only relative paths work at the moment.
-
-
+.. note::
+   The following examples assume that the `BORG_REPO` and `BORG_PASSPHRASE` environment variables are set
+   if the repo or passphrase is not explicitly given.
 
+- Borg does not crash if called with ``borg``
+- ``borg init --encryption repokey-blake2 ./demoRepo`` runs without an error/warning.
+  Note that absolute paths only work if the protocol is explicitly set to file://
```

Comment:
why is that?

Reply:
wouldn't a path like /foo/bar also work?

Reply:
That is not absolute on windows.

Reply:
Windows absolute paths like "C:\MyDocs" are recogniced as remote repositories, using the file protocol explicit prevents this.

Reply:
And `/MyDocs` ? Assuming that `C:` is the current drive?

Reply:
If C: is current drive `\MyDocs` would mean `C:\MyDocs` so `/MyDocs` should pass and work.
If C: is current drive how would I backup `X:\foo`, `file://X:\foo`?

Reply:
Adding drive letter support is a good topic for a separate commit that only deals with that.